### ISSUE TITLE
Can't create an event with service request comment

### ISSUE THREAD

**Issue by JChipault** [USGS-WiM/whispers#1064]
**Describe the bug**
When creating an event that has a service request comment, get this error message when try to save.

![image](https://user-images.githubusercontent.com/49287076/95906922-6526df80-0d60-11eb-9f47-132964bb117e.png)

User is not able to select comment type during create event. I think the system is supposed to auto-assign the comment type.

See -  https://github.com/USGS-WiM/whispersservices/issues/253

**To Reproduce**
1. click Create Event
2. fill in all required fields
3. select a "request diagnostic and consultative services" or "request consultative services ONLY"
4. type comment
5. click "Save Event"

**Expected behavior**
Should save event while auto-assigning comment type of "Diagnostic" to any service request comments

**Desktop (please complete the following information):**
Happened in Firefox and Chrome

**Additional context**
Neil looked in the tables and thinks that the last time that a service request comment was created was on 8.27.2020 so this bug seems to have popped up since then.

Issue happened on both the notifications test site AND the normal test site. Have not checked the live site; could you make sure that the comment type for service requests are auto-assigning on the live site?

**Comment by BlakeDraper** [comment]
@aaronstephenson can you take a look at this on your end? As Jenny explains above, comment type for a service request comment is implicit and should be assigned by the services on processing. 

Below is the submitted JSON, as a POST to `/whispersservices/events/`

works fine on whispersdev, but neither new VM or old VM test envs.

```
{
  "event_reference": "",
  "event_type": "1",
  "complete": false,
  "public": true,
  "staff": null,
  "event_status": "1",
  "legal_status": 1,
  "legal_number": "",
  "new_organizations": [
    2
  ],
  "new_service_request": {
    "request_type": 1,
    "new_comments": [
      {
        "comment": "test comment for bug"
      }
    ]
  },
  "new_event_diagnoses": [],
  "new_comments": [],
  "new_eventgroups": [],
  "new_event_locations": [
    {
      "name": "",
      "start_date": "2020-10-02",
      "end_date": "2020-10-14",
      "country": "30",
      "administrative_level_one": 51,
      "administrative_level_two": 3055,
      "latitude": null,
      "longitude": null,
      "land_ownership": "4",
      "gnis_name": "",
      "gnis_id": "",
      "site_description": "public park",
      "history": "",
      "environmental_factors": "",
      "clinical_signs": "",
      "comment": "",
      "new_location_species": [
        {
          "species": 247,
          "population_count": null,
          "sick_count": null,
          "dead_count": 3,
          "sick_count_estimated": null,
          "dead_count_estimated": null,
          "priority": null,
          "captive": false,
          "age_bias": null,
          "sex_bias": null,
          "new_species_diagnoses": []
        }
      ],
      "new_location_contacts": []
    }
  ],
  "new_read_collaborators": [],
  "new_write_collaborators": [],
  "quality_check": null
}
```

**Comment by aaronstephenson** [comment]
Fixed in whispersservices [9e486df](https://github.com/USGS-WiM/whispersservices/commit/9e486df9637d52a9240d5d62335acb8c8a95d258)
